Modules/Quotas/src/Quotas.Application/Identities/Queries/GetIdentity/Handler.cs

```diff
@@ -1,18 +1,21 @@
 ﻿using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
 using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
 using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Metrics;
 using MediatR;
 
 namespace Backbone.Modules.Quotas.Application.Identities.Queries.GetIdentity;
 public class Handler : IRequestHandler<GetIdentityQuery, GetIdentityResponse>
 {
     private readonly IIdentitiesRepository _identitiesRepository;
     private readonly IMetricsRepository _metricsRepository;
+    private readonly MetricCalculatorFactory _metricCalculatorFactory;
 
-    public Handler(IIdentitiesRepository identitiesRepository, IMetricsRepository metricsRepository)
+    public Handler(IIdentitiesRepository identitiesRepository, IMetricsRepository metricsRepository, MetricCalculatorFactory factory)
```

Comment:
We should probably rename this.
```suggestion
    public Handler(IIdentitiesRepository identitiesRepository, IMetricsRepository metricsRepository, MetricCalculatorFactory metricCalculatorFactory)
```